src/pages/index.astro

```diff
@@ -158,4 +158,16 @@ const examples = (await getCollection('example'));
       </Card>
     </div>
   </Section>
+
+  <!-- CNCF & LF -->
+  <Section class="bg-base-300">
+    <h2 class="text-primary text-3xl font-bold text-center mb-4">Proudly Open Source</h2>
+    <p class="text-center">
+      Serverless Workflow is an open-source project under the governance of the <a class="link link-accent" href="https://www.cncf.io/" target="_blank"  rel="noopener">Cloud Native Computing Foundation (CNCF)</a> and a proud part of the <a class="link link-accent" href="https://www.linuxfoundation.org/" target="_blank"  rel="noopener">Linux Foundation</a>.
```

Comment:
Same comment than above regarding: `and a proud part of the`

I'd personally remove `proud`

Reply:
Done